hat/app/org/hatdex/hat/api/controllers/DataDebits.scala

```diff
@@ -48,140 +53,242 @@ class DataDebits @Inject() (
     dataDebitService: DataDebitService,
     loggingProvider: LoggingProvider,
     implicit val ec: ExecutionContext,
-    implicit val applicationsService: ApplicationsService) extends HatApiController(components, silhouette) with RichDataJsonFormats {
+    implicit val applicationsService: ApplicationsService)
+    extends HatApiController(components, silhouette)
+    with RichDataJsonFormats {
 
   private val logger = loggingProvider.logger(this.getClass)
 
   def registerDataDebit(dataDebitId: String): Action[DataDebitSetupRequest] =
-    SecuredAction(WithRole(Owner(), DataDebitOwner(""), Platform()) || ContainsApplicationRole(Owner(), DataDebitOwner(""), Platform())).async(parsers.json[DataDebitSetupRequest]) { implicit request ⇒
-      dataDebitService.createDataDebit(dataDebitId, request.body, request.identity.userId)
-        .andThen(dataEventDispatcher.dispatchEventDataDebit(DataDebitOperations.Create()))
-        .map(debit ⇒ Created(Json.toJson(debit)))
+    SecuredAction(
```

Comment:
Oh, we'll need to look into scalafmt config for lines 63-68 refactor.
This

break

up 

is 

not

readable.
Not blocking the PR for this though.

Reply:
I
Don't
See
The
Problem.  

If you can find the config to suppress this change I will add it.  

I don't mind either format. :)